test/test_collections.rb

```diff
@@ -186,6 +186,9 @@ class TestCollections < JekyllUnitTest
     end
 
     should "include the symlinked file from site.source in the list of docs" do
+      # no support for including symlinked file on Windows
+      skip_if_windows
```

Comment:
Then here I'd be specific with my message and say `skip_if_windows, "Jekyll does not currently support symlinsk on Windows."`.